### ISSUE TITLE
Support Love Live! School idol festival 2 MIRACLE LIVE

### ISSUE THREAD

**Issue by ghost** [nosyrbllewe/LLASDecryptor#11]
Hi, do you plan to support Love Live! School idol festival 2 MIRACLE LIVE?

**Comment by nosyrbllewe** [comment]
No, I do not and haven't really looked into decrypting it personally. 
What files did you want to get out of it? Songs? Beatmaps? Cards?

**Comment by ghost** [comment]
Just the songs
